treenode.go

```diff
@@ -126,9 +128,9 @@ func (n *TreeNodeInstance) SendTo(to *TreeNode, msg interface{}) error {
 	var c *GenericConfig
 	// only sends the config once
 	n.configMut.Lock()
-	if n.config != nil {
+	if !n.sentTo[to.ID] {
 		c = n.config
-		n.config = nil
+		n.sentTo[to.ID] = true
 	}
```

Comment:
And now your question has made me think about other issues... So usually, we *assume* the first message MUST contains the configuration. But is that really what we want all the time ? It can probably happen that a config has to be sent to all *children* and then in case stuff goes wrong, the root contact the sub-children (for example, what our theoretical CoSi would likely to if it were to exist). The message for the sub-children should *not* contain the config. 

In any way, this specific case is already handled at the recipient level, where the Overlay will only look for the config if the TreeNode already exists, but that means the config received from the root (to the subchildren) will stay in memory forever... :/

For now let's move on with this approach but we'll have to dig more afterwards.  @christophebertrand if you think of stg, don't hesitate to chim in ;)